tests/Modifier/DiscountPriceTest.php

```diff
@@ -1,25 +1,29 @@
 <?php
+namespace Blesta\Pricing\Tests\Unit\Modifier;
```

Comment:
Namespace should be `Blesta\Pricing\Tests\Modifier;` *OR* test should be moved into a `Unit` directory within `tests/` (notice case).